tests/cli-tests/compression/window-resize.sh.stderr.glob

```diff
@@ -0,0 +1 @@
+...
```

Comment:
minor : I'm wondering what happens if this file is not provided, or if it is replaced by the `*.ignore` suffix variant.
This could be a more explicit way to express that `stderr` output is not significant.
cc @terrelln .

Reply:
Using the `.ignore` suffix seems to work! I can switch to that, I like that it is a bit more explicit than redirection.